src/angular-moment-picker.js

```diff
@@ -169,7 +171,10 @@
 					return !angular.isDefined($scope.maxDateMoment) || value.isBefore($scope.maxDateMoment, precision) || value.isSame($scope.maxDateMoment, precision);
 				},
 				isSelectable: function (value, precision) {
-					return $scope.limits.isAfterOrEqualMin(value, precision) && $scope.limits.isBeforeOrEqualMax(value, precision);
+					var selectable = angular.isFunction($scope.$parent[$scope.selectable]) ? $scope.$parent[$scope.selectable] : function () {
```

Comment:
Keep in mind that you should never use `$parent` scope. The directive uses a private scope, so you should care only about your scope (`$scope`).

When you define an attribute using `&` binding ([read more](https://docs.angularjs.org/api/ng/service/$compile#-scope-)) you can call the property directly specifying any attributes you want to pass through to your callback as an object. In other terms, you should update the code like this:

```javascript

isSelectable: function (value, precision) {
	var selectable = angular.isFunction($scope.selectable) ? $scope.selectable({ value: value, precision: precision }) : true;
	return $scope.limits.isAfterOrEqualMin(value, precision) && $scope.limits.isBeforeOrEqualMax(value, precision) && selectable;
}
```